### DISCUSSION TITLE
the updated repo for deeplearning

### DISCUSSION THREAD

**Discussion by BWhannou** [ML-for-B-E/.github#7] [Q&A] [answered]
@akouete-kpakpo @zolatoi where is the repo/folder with all your work on deep learning supports?

**Comment by akouete-kpakpo** [comment, marked as answer]
@BWhannou you have to switch on branch `11-deep-learning-yolo_v5`. The last demo notebook has not yet been merged yet.

## Reference
https://github.com/ML-for-B-E/summer-camp-2021/tree/11-deep-learning-yolo_v5/cours_de_stats
